packages/css/src/components/mark/mark.scss

```diff
@@ -0,0 +1,8 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+.amsterdam-mark {
```

Comment:
There apears to be some inline-padding to the mark if you compare it to Figma, but you'll never get the same padding on a new line if the mark breaks, so probably better to leave it default.

Reply:
I agree.